test/autoupdate.test.js

```diff
@@ -0,0 +1,174 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback-connector-couchdb2
+// This file is licensed under the Apache License 2.0.
+// License text available at https://opensource.org/licenses/Apache-2.0
+
+'use strict';
+
+/*
+NOTE: This suite is also used by loopback-connector-cloudant.
+To support Optional Test Opt-In/Out, names of tests to be skipped
+can be added to skips array. All tests must start with a check to
+see if the test should be skipped or not.
+The following line can be used to accomplish this:
+
+this.test.pending ? this.skip() : null;
```

Comment:
After https://github.com/strongloop/loopback-connector-couchdb2/commit/a37914285b696ed3a9a807b3131a4226b9861f57 checked in we don't need this line anymore I think...